personal.html

```diff
@@ -11,82 +11,55 @@
+                    alt="Jay-Z">
+                    <p>Beyoncé has been married to Jay-Z since April 4, 2008. He has been widely acknowledged as one of the
+                    greatest rappers of all time.
+                </p>
+            </section>
+            <section id="Blue_Ivy">
+                <h3>Oldest Daughter</h3>
+                <img class="family_pic" id="daughter_pic" src="https://www.thefamouspeople.com/profiles/images/blue-ivy-carter-1.jpg"
+                    alt="daughter">
+                <p>Blue Ivy Carter was born on January 7, 2012, at Lenox Hill hospital in New York.
+                </p>
+            </section>
 
-        </section>
-        <h3>Husband</h3>
-        <section class ="personalInfo" id="husband">
-            Beyoncé has been married to Jay-Z since April 4, 2008. 
-            <img id ="Jay-Z_pic" src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/9c/Jay-Z_%40_Shawn_%27Jay-Z%27_Carter_Foundation_Carnival_%28crop_2%29.jpg/361px-Jay-Z_%40_Shawn_%27Jay-Z%27_Carter_Foundation_Carnival_%28crop_2%29.jpgZ" alt="Jay-Z">
-            <footer>
-
-            </footer>
-            <br>
-            <br>
-            <br>
-            <br>
-            <br>
-            <br>
-            <br>
-            <br>
-            <br>
-            <br>
-            <br>
-            <br>
-            <br>
-        </section>
-        <h3>Oldest Daughter</h3> 
-        <section class = "familyMembers" id="firstDaughter">
-Blue Ivy Carter was born on January 7, 2012, at Lenox Hill hospital in New York. 
+            <section id="Twins">
+                <h3>Twins</h3>
+                <img class="family_pic" id="twins_pic" src="http://static3.uk.businessinsider.com/image/5a087ce8ec1ade38242974bd-480/beyonce-twins-photo.png"
+                    alt="twins">
+                <p> On June 13, 2017, Beyonce gave birth to twins, Rumi and Sir Carter. The twins were delivered by Dr
+                    Paul Crane at Ronald Reagan UCLA Medical Center in Los Angeles.</p>
+            </section>
 
-<img id ="daughter_pic" src="https://www.thefamouspeople.com/profiles/images/blue-ivy-carter-1.jpg" alt="daughter">
-
-        </section>
-        <br>
-        <br>
-        <br>
-        <br>
-        <br>
-        <br>
-        <br>
-        <br>
-        <br>
-        <br>
-        <br>
-        <br>
-        <br>
-
-        <h3>Twins</h3> 
-        
-             
-             
-             <section class ="familyMembers" id="twins"></section>
-                <img id ="twins_pic" src="https://www.essence.com/wp-content/uploads/2018/07/1532613703/beyonce%20twins-445x297.jpeg" alt="twins">           
-                          
-            On June 13, 2017, Beyonce gave birth to twins, Rumi and Sir Carter. The twins were delivered by Dr Paul Crane at Ronald Reagan UCLA Medical Center in Los Angeles. 
     </article>
 
-
+    </div>
 </body>
+<footer>&copy; Copyright 2019 HTML.am</footer>
```

Comment:
Please remove the "&copy; Copyright 2019 HTML.am" on line 60 and add the closing footere tag at the end of line 61.